thunderbird/thunderbird.dockerfile

```diff
@@ -22,13 +22,14 @@ RUN sudo chown user:user /home/user/thunderbird/.mozconfig
 
 # Set up Mercurial so mach doesn't complain.
 RUN mkdir -p /home/user/.mozbuild \
- && ./mozilla/mach mercurial-setup -u
+ && ln -nsf /home/user/thunderbird/mach /home/user/bin/mach \
```

Comment:
Hm, I'm not sure `/home/user/bin` is actually in the `PATH`. But a working CI will tell us that. If it's not, we can always add it, or do something like `sudo ln [...] /usr/local/bin/mach`.